Pursuit-Core-Web-Postgres-Lab/assets/facebook_db.sql

```diff
@@ -0,0 +1,78 @@
+--Database & Table Creation;
+
+DROP DATABASE IF EXISTS facebook_db;
+CREATE DATABASE facebook_db;
+
+\c facebook_db;
+
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS posts;
+DROP TABLE IF EXISTS likes;
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    name TEXT,
+    age INTEGER
+);
+
+CREATE TABLE posts (
+    id SERIAL PRIMARY KEY,
+    poster_id INT REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
Remember that there is also `ON DELETE SET NULL`

Reply:
Try and think about both and see which one would fit better for each case scenario